src/settings.js

```diff
@@ -53,3 +52,17 @@ export default {
   PORT: 5683,
   HOST: "localhost",
 };
+
+const testSettings = {
```

Comment:
I don't think this should go here.  Any way you can have this in the tests directory and just override settings?